classification/profiles/databricks.yaml

```diff
@@ -0,0 +1,31 @@
+# [Recommended] Uncomment fields below to set an MLflow experiment to track the recipe execution.
+# If unset, a default experiment based on runtime context will be created.
+# experiment:
+  # name: "/Shared/sklearn_classification_experiment"
+
+# Set the registry server URI. This property is especially useful if you have a registry
+# server that’s different from the tracking server.
+# Profile could be created using https://github.com/databricks/databricks-cli#installation
+model_registry:
+  # registry_uri: "databricks://PROFILE_NAME"
+  # FIXME::REQUIRED: Specifies the name of the Registered Model to use when registering a trained
+  #                 model to the MLflow Model Registry.
+  model_name: "credit_card_fraud_classifier"
+
+# Override the default train / validation / test dataset split ratios
+SPLIT_RATIOS: [0.75, 0.125, 0.125]
+
+INGEST_CONFIG:
+  # For different options please read: https://github.com/mlflow/mlp-classification-template#ingest-step
+  using: parquet
+  location: "dbfs:/databricks-datasets/credit-card-fraud/data/part-00000-tid-898991165078798880-9c1caa7b-283d-47c4-9be1-aa61587b3675-0-c000.snappy.parquet"
```

Comment:
there are several data files, this is just one of them - can we use a regex to ingest the entire dataset